src/libraries/Native/Unix/System.Native/browser/pal_entropy_bridge.js

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We do not have other OS-specific System.Native implementation in subdirectories. For example, `pal_log.m` is Apple OS specific and it does not live in subdirectory.

It would be nice to maintain same style.

Reply:
renamed and addressed in recent commits